src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -6318,303 +6359,332 @@ namespace EnergyPlus::OutputReportTabular {
-                            (SELECT_CASE_var == iAggType::SumOrAverageHoursShown)) {
-                            ++columnUsedCount;
-                        } else if ((SELECT_CASE_var == iAggType::Maximum) || (SELECT_CASE_var == iAggType::Minimum) ||
-                                   (SELECT_CASE_var == iAggType::MaximumDuringHoursShown) || (SELECT_CASE_var == iAggType::MinimumDuringHoursShown)) {
-                            columnUsedCount += 2;
-                        }
-                    }
-                } // jColumn
-                columnHead.allocate(columnUsedCount);
-                columnWidth.dimension(columnUsedCount, 14); // array assignment - same for all columns
-                tableBody.allocate(columnUsedCount, 16);
-                tableBody = ""; // set entire table to blank as default
-                columnRecount = 0;
-                for (kColumn = 1; kColumn <= ort->MonthlyTables(curTable).numColumns; ++kColumn) {
-                    curCol = kColumn + ort->MonthlyTables(curTable).firstColumn - 1;
-                    curAggString = aggString.at(ort->MonthlyColumns(curCol).aggType);
-                    if (len(curAggString) > 0) {
-                        curAggString = " {" + stripped(curAggString) + '}';
-                    }
-                    // do the unit conversions
-                    if (ort->unitsStyle == iUnitsStyle::InchPound) {
-                        varNameWithUnits = ort->MonthlyColumns(curCol).varName + unitEnumToStringBrackets(ort->MonthlyColumns(curCol).units);
-                        LookupSItoIP(state, varNameWithUnits, indexUnitConv, curUnits);
-                        GetUnitConversion(state, indexUnitConv, curConversionFactor, curConversionOffset, curUnits);
-                    } else { // just do the Joule conversion
-                        // if units is in Joules, convert if specified
-                        if (UtilityRoutines::SameString(unitEnumToString(ort->MonthlyColumns(curCol).units), "J")) {
-                            curUnits = energyUnitsString;
-                            curConversionFactor = energyUnitsConversionFactor;
-                            curConversionOffset = 0.0;
-                        } else { // if not joules don't perform conversion
-                            curUnits = unitEnumToString(ort->MonthlyColumns(curCol).units);
-                            curConversionFactor = 1.0;
-                            curConversionOffset = 0.0;
+        for (int iUnitSystem = 0; iUnitSystem <= 1; iUnitSystem++) {
+            iUnitsStyle unitsStyle_cur = ort->unitsStyle;
+            bool produceTabular = true;
+            bool produceSQLite = false;
+            if (produceDualUnitsFlags(iUnitSystem, ort->unitsStyle, ort->unitsStyle_SQLite, unitsStyle_cur, produceTabular, produceSQLite)) break;
+
+            // set the unit conversion
+            // if (ort->unitsStyle == iUnitsStyle::None) {
```

Comment:
Please remove this commented out code

Reply:
Ok. Will do the clean up in some places.

Reply:
This one is done.